OSDOCS-9583 (#72868)
openshift/openshift-docs: modules/rosa-accessing-your-cluster-quick.adoc

W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
I: Admin account has been added to cluster 'cluster_name'. It may take up to a minute for the account to become active.
I: To login, run the following command:

> oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \ <1>

Reviewer: Unless this is intentionally misspelled, I think there's a typo here:
```suggestion
oc login https://api.cluster-name.t6k4.i1.organization.org:6443 \ <1>
```

Author: Created a Jira: https://issues.redhat.com/browse/OSDOCS-9968 to implement out of scope comments for this PR. Thank you!

> oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \ #<1>

Reviewer 2: You can remove # from [source,terminal] samples as it is not used with call outs in [source,terminal].

See: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks

--username cluster-admin \
--password FWGYL-2mkJI-3ZTTZ-rINns
----

> <1> For {hcp-title-first} cluster, the port number should be `443`.

Reviewer 2: Should this read as "For {hcp-title-first} clusters" or "For a {hcp-title-first} cluster"?

Author: > Should this read as "For {hcp-title-first} clusters" or "For a {hcp-title-first} cluster"?
"For a {hcp-title-first} cluster" looks good to me for here and future instances for this PR. I will make the updates accordingly for this PR.

> Also notice sometimes it's {hcp-title-first} and sometimes it's {hcp-title}...which I now see is because {hcp-title-first} is the first mention so it's the full name, and {hcp-title} is a subsequent mention.
Yes, exactly.

. Enter the `oc login` command, username, and password from the output of the previous command:

+
.Example output
[source,terminal]
----

> $ oc login https://api.cluster_name.t6k4.i1.oragnization.org:6443 \ #<1>

Reviewer 2: You can remove # from [source,terminal] samples as it is not used with call outs in [source,terminal].

See: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks

> $ oc login https://api.cluster_name.t6k4.i1.oragnization.org:6443 \ <1>

Reviewer: Unless this is intentionally misspelled, I think there's a typo here:
```suggestion
$ oc login https://api.cluster_name.t6k4.i1.organization.org:6443 \ <1>
```

Author: Created a Jira: https://issues.redhat.com/browse/OSDOCS-9968 to implement out of scope comments for this PR. Thank you!

>  --username cluster-admin \
>  --password FWGYL-2mkJI-3ZTTZ-rINns

> Login successful.                                                                                                                                                                                                                                                       You have access to 77 projects, the list has been suppressed. You can list all projects with ' projects'

Reviewer: This is out of scope for your PR, but is the line that starts with "You have access to 77 projects..." supposed to appear on a new line in the terminal? In the rendered doc, it appears on the same line as "Login successful" but very far to the right, causing this code block to have a horizontal scroll because of its width. 
```suggestion
Login successful.                                                                                                                                                                                                                                                       
You have access to 77 projects, the list has been suppressed. You can list all projects with ' projects'
```
Up to you whether you want to fix this, if applicable, since this isn't something you changed.

Author: Created a Jira: https://issues.redhat.com/browse/OSDOCS-9968 to implement out of scope comments for this PR. Thank you!

> Login successful.                                                                                                                                                                                                                                                       You have access to 77 projects, the list has been suppressed. You can list all projects with ' projects'

Reviewer: On a similar note and also up to you, but seems like there's an extra space here:
```suggestion
Login successful.                                                                                                                                                                                                                                                       You have access to 77 projects, the list has been suppressed. You can list all projects with 'projects'
```

Author: Created a Jira: https://issues.redhat.com/browse/OSDOCS-9968 to implement out of scope comments for this PR. Thank you!

----

> <1> For {hcp-title} cluster, the port number should be `443`.

Reviewer 2: Should this read as "For {hcp-title} clusters" or "For a {hcp-title} cluster"?

Might also consider rephrasing for clarity. Something like: "For {hcp-title} clusters, use port 443."

Author: Please see [my comment](https://github.com/openshift/openshift-docs/pull/72868#discussion_r1524263756).
